### PR TITLE
Sparkbox sprint 23

### PR DESCRIPTION
**Pull request by kaseybon**
# READY FOR REVIEW

## Summary
- feat (STN-373): added lando and CSS documentation for developers (#671)
- feat (STN-481): update the default color pairing for Colorful to Cardinal (#685)
- fix (STN-492): secondary nav header spacing and border-top (#684)
- fix (STN-480): Fix spacing beside navigation dropdown toggle (#686)
- feat (STN-495): Add the Warbler Color Pairing (#688)
- feat (STN-490): refactor paragraph type grid system to use CSS Grid (#689) 
- feat (STN-392): adding behat tests (#658) 
- feat (STN-493): add title region to 3 column w/image layout (#692)
- feat (STN-522): create utility class to display title text on Colorful (#696)
- fix (STN-520): spacing bug fix for hb-views-divider class (#697)

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Ensure Code Climate passes against the `develop` branch. Link here: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/691
- [x] Assign to Mike for review.

## Need Review By (Date)
8/25

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
1. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ three_column_w_image:
   icon_map:
     - [sidebar_first, main, sidebar_second]
   regions:
+    title:
```
Comment (pookmish) [inline docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml:27]: This should have a respective change to the template in the layout module as well.

**Review by pookmish — CHANGES_REQUESTED** [review]
Just a small addition is necessary.

File: docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml (hunk 1 above)
Comment (kaseybon) [reply docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml:27]: @pookmish This should be good to go now.

Comment (pookmish) [reply docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml:27]: Thank you

**Pull request merged**